sources/packages/backend/libs/sims-db/src/entities/student.model.ts

```diff
@@ -110,4 +116,14 @@ export class Student extends RecordDataModel {
     nullable: true,
   })
   disabilityStatusEffectiveDate?: Date;
+
+  /**
+   * Student supplier information data from the integration with Corporate Accounting System (CAS).
+   */
+  @OneToOne(() => CASSupplier, { eager: false, cascade: true, nullable: true })
```

Comment:
Please change the cascade to be specific as `cascade: ["insert", "update"]`.
The most recent decision was to be as specific as possible.